### DISCUSSION TITLE
Icons

### DISCUSSION THREAD

**Discussion by CatInpringels** [prasanthrangan/hyprdots#676] [Q&A]
How do I make it so that the Decay theme has the icons of the mocha theme? really basic question i know but i cant find any way to do it, ive tried moving around some theme configs but doesnt seem to work.

**Comment by ghost** [comment]
Looked into this, seems that `~/.config/hypr/themes` is the directory responable for managing the gtk-theme, icon-set, and color-scheme. If you open both the `Decay-Green.conf` file and the `Catppuccin-Mocha.conf` file you'll see that the icon-theme is different, so if you copy the icon-theme for Catppuccin-Mocha to Decay-Green, it should work when you apply it!
